volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -994,6 +995,81 @@ def privileges(self):
                 vollog.log(constants.LOGLEVEL_VVVV, "Broken Token Privileges.")
 
 
+class KTIMER(objects.StructType):
+    """A class for Kernel Timers"""
+
+    VALID_TYPES = {
+        8: "TimerNotificationObject",
+        9: "TimerSynchronizationObject",
+    }
+
+    def get_signaled(self):
+        if self.Header.SignalState:
+            return "Yes"
+        return "-"
+
+    def get_raw_dpc(self):
+        """Returns the encoded DPC since it may not look like a pointer after encoding"""
+        symbol_table_name = self.get_symbol_table_name()
+        ulonglong_type = self._context.symbol_space.get_type(
+            symbol_table_name + constants.BANG + "unsigned long long"
```

Comment:
There is a question as to whether this type will always exist (recent versions of clang call it something like `long unsigned int` I think?).  It's probably fine and I don't know what else we'd do (discussion is still going on at #1041 ).  If this is a pointer though, then it should be a pointer which will get the size right automatically.

Reply:
changed to pointer in latest commit